src/scportrait/pipeline/extraction.py

```diff
@@ -462,50 +476,21 @@ def _save_removed_classes(self, classes):
             f"A total of {len(classes)} cells were too close to the image border to be extracted. Their cell_ids were saved to file {filtered_path}."
         )
 
-    def _save_cell_info(self, save_index, cell_id, image_index, label_info, stack):
-        """helper function to save the extracted cell information to the temporary datastructures
-
-        Parameters
-        ----------
-        save_index : int
-            index location in the temporary datastructures where the cell in question needs to be saved
-        cell_id : int
-            unique identifier of extracted cell
-        image_index : int | None
-            index of the source image that was processed. Only relevant for TimecourseProjects. Otherwise None.
-        label_info : str | None
-            additional information that is to be saved with the extracted cell. Only relevant for TimecourseProjects. Otherwise None.
-        stack : np.array
-            extracted single cell images that are too be saved
+    def _extract_classes(self, arg, return_results: bool = False):
         """
-        # label info is None so just ignore for the base case
-        # image_index is none so just ignore for the base case
-
-        # save single cell images
-        self._tmp_single_cell_data[save_index] = stack
-        self._tmp_single_cell_index[save_index] = [save_index, cell_id]
-
-    def _save_failed_cell_info(self, save_index, cell_id, image_index, label_info):
-        """save the relevant information for cells that are too close to the image edges to extract
-
-        Parameters
-        ----------
-        save_index : int
-            index location in the temporary datastructures where the cell in question should have been saved, this index
-            location will later be deleted
-        cell_id : int
-            unique identifier of the cell which was unable to be extracted
+        Processing for each individual cell that needs to be run for each center.
 
-        """
+        Args:
+            px_center (tuple): The pixel center of the cell.
+            arg (tuple): The arguments that are passed to the function.
```

Comment:
Which function?

Reply:
addressed by improving the docstring and making this explicitly clear